kong.conf.default

```diff
@@ -251,24 +289,34 @@
                                  # to open a new connection.
 
 #headers = server_tokens, latency_tokens
-                                 # Specify Kong-specific headers that should
-                                 # be injected in responses to the client.
-                                 # Acceptable values are:
-                                 # - `server_tokens`: inject 'Via' and 'Server'
-                                 #   headers.
-                                 # - `latency_tokens`: inject
-                                 #   'X-Kong-Proxy-Latency' and
-                                 #   'X-Kong-Upstream-Latency' headers.
-                                 # - `X-Kong-<header-name>`: only inject this
-                                 #   specific the header when applicable.
-                                 #
-                                 # Example:
-                                 #   headers = via, latency_tokens
-                                 #
-                                 # This value can be set to `off`, which
```

Comment:
This was more readable as a separate note vs. a bullet point like other accepted values as of this patch.